docs/docs/noir/standard_library/containers/boundedvec.md

```diff
@@ -294,6 +294,43 @@ Example:
 
 #include_code bounded-vec-map-example noir_stdlib/src/collections/bounded_vec.nr rust
 
+### mapi
+
+```rust
+pub fn mapi<U, Env>(self, f: fn[Env](u32, T) -> U) -> BoundedVec<U, MaxLen>
+```
+
+Creates a new vector of equal size by calling a closure on each element in this
+vector, along with its index in the vector.
+
+Example:
+
+#include_code bounded-vec-mapi-example noir_stdlib/src/collections/bounded_vec.nr rust
+
+### `for_each`
```

Comment:
I assume the underscore messes up the formatting here without quoting this? Would a `\` fix it?